tests/functional/intelligent-host-selection/01-periodic-clear-badhosts/flow.cylc

```diff
@@ -0,0 +1,18 @@
+
+[meta]
+title = "Try out scenarios for intelligent host selection."
+description = """
+Tasks
+- mixedhost contains some hosts that will and won't fail
+"""
+
+[scheduling]
+    cycling mode = integer
+    initial cycle point = 1
+    [[graph]]
+        R1 = mixedhosttask
+
+[runtime]
+    [[mixedhosttask]]
+        platform = mixedhostplatform
+        script = true
```

Comment:
No newline at EOF, don't know if that matters

Reply:
It should be there, thanks for pointing it out. It doesn't matter in the grand scheme of things, but I'd like what I write to look more professional and this sort of thing helps. 👍🏼 
Thanks for spotting.